src/utils/record-completeness/ComputeRecordCompleteness.ts

```diff
@@ -62,7 +62,8 @@ export const computeRecordCompleteness = (techRecordWrapper: ITechRecordWrapper)
     systemNumber: techRecordWrapper.systemNumber,
     vin: techRecordWrapper.vin,
     primaryVrm: techRecordWrapper.primaryVrm,
-    trailerId: techRecordWrapper.trailerId
+    // FIXME: handle trailerId in a better way
```

Comment:
To handle the record completeness in a better way, you would need to create a separate checker for each vehicle type?

Reply:
I think this should be done a separate iterative step. This is a complex logic and I don't want it to be a part of the current process.